src/qt/pivx/settings/settingsconsolewidget.h

```diff
@@ -44,8 +44,8 @@ class SettingsConsoleWidget : public PWidget
 
 public Q_SLOTS:
     void clear(bool clearHistory = true);
-    void message(int category, const QString &msg) { message(category, msg, false); }
-    void message(int category, const QString &message, bool html);
+    void response(int category, const QString &message) { messageInternal(category, message); };
```

Comment:
non-blocking nit: 
let's call this `reply` instead of `response` (for consistency with the `RPCExecutor` signal)